examples/resources/awscc_elasticloadbalancingv2_target_group/elasticloadbalancingv2_target_group_ip.tf

```diff
@@ -0,0 +1,11 @@
+resource "awscc_elasticloadbalancingv2_target_group" "ip-example" {
+  name     = "ip-example"
+  port     = 80
+  protocol = "HTTP"
+  target_type = "ip"
+  vpc_id   = awscc_ec2_vpc.main.id
+  tags = [{
+    key   = "Modified By"
+    value = "AWSCC"
+  }]
+}
```

Comment:
run `terraform fmt` to fix indentation

Reply:
I have rerun `terraform fmt` on the resources file, thanks for noticing.